prep-javascript-dom-events/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Events</title>
+</head>
+
+<body>
+  <button class="hello-world-btn">Hello World!</button>
+  <script>
+    // Your code here
+
+    document.addEventListener('DOMContentLoaded', funcDocument);
+    function funcDocument(func1) {
+      return func1;
+    }
+    var loaded = funcDocument('Content Loaded!');
+    console.log(loaded);
```

Comment:
Hi Kevin, try simplifying this part a bit by logging your string directly inside of the `funcDocument` function.

```js
document.addEventListener('DOMContentLoaded', funcDocument);
function funcDocument() {
  console.log('Content Loaded!');
}
```

Reply:
![prep-dom-events-correction](https://user-images.githubusercontent.com/43438792/150653204-56038e3f-fd78-4697-9a6e-b2ce469fe491.png)